code/modules/mob/dead/observer/observer.dm

```diff
@@ -602,12 +602,7 @@ This is the proc mobs get to turn into a ghost. Forked from ghostize due to comp
 /mob/dead/observer/verb/view_manifest()
 	set name = "View Crew Manifest"
 	set category = "Ghost"
-
-	var/dat
-	dat += "<h4>Crew Manifest</h4>"
-	dat += GLOB.data_core.get_manifest(OOC = TRUE)
```

Comment:
And this begins the complication.

Here the OOC manifest is pulled because it shows SSDs in lobby and ghost